services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -70,7 +73,7 @@ def reset
       teacher = find_or_create_teacher_data(row['Teacher'], school, subscription)
 
       # create classroom data
-      classroom = Classroom.create(name: row['Classroom'])
+      classroom = Classroom.create(name: row['Classroom'], grade: (GRADE_MIN..GRADE_MAX).to_a.sample.to_s)
```

Comment:
You could extract out another constant:
`GRADES = (GRADE_MIN..GRADE_MAX).to_a`

Then add a private method:

`private def sample_grade = GRADES.sample`

`classroom = Classroom.create(name: row['Classroom'], grade: sample_grade)`